### ISSUE TITLE
Getting Raw Content

### ISSUE THREAD

**Issue by net-brain** [tinymce/tinymce-blazor#9]
Is there a way to get the raw content? Value seems to be the HTML content. In TinyMce there is a parameter for the GetContent() Method, but I cannot find one for blazor.

**Comment by jscasca** [comment]
Hi @net-brain , what do you mean by the raw content? Do you mean the Text content?

It certainly feels like the wrapper needs a way to specify the format. Maybe we need to put a `GetText()`? Or maybe a `GetContent(string format)`?

**Comment by net-brain** [comment]
Hi @jscasca , Yes, I mean the text. I would like to use it for a wiki and I need it to save and do some db oriented things with the text (e.g. full text index). I think GetText is better than parameter for the GetContent.

On the other side: I searched for components doing that in c#. I'm not sure where the functionality should be. 

BTW: Great component! I really appreciate your work!

**Comment by jscasca** [comment]
So #12 adds the readonly Text parameter that allows users to either get the Text or bind the text output as follows:
```
<TinyMCE.Blazor.Editor @ref="Editor" @bind-Text="textVal" />
...
private string textVal = ""; // bind to var
string content = Editor.Text; // get property
```
Hope to have this merged soon

**Comment by jscasca** [comment]
@net-brain This has just been released in v0.0.4

Glad you like the component!
